src/controller/CourseController.ts

```diff
@@ -22,16 +22,10 @@ export class CourseController {
                         where[param] = Raw(alias => `LOWER(${alias}) LIKE '%${value.toLowerCase()}%'`);
                         break;
                     case 'number':
-                        const hasMin = value.hasOwnProperty('min');
-                        const hasMax = value.hasOwnProperty('max');
-                        if ( hasMin && hasMax ) {
-                            where[param] = Between(value.min, value.max);
-                        } else if (hasMax) {
-                            where[param] = LessThanOrEqual(value.max);
-                        } else if (hasMin) {
-                            where[param] = MoreThanOrEqual(value.min);
+                        if (Array.isArray(value)) {
+                            where[param] = Between(value[0], value[1]);
                         } else {
-                            where[param] = Equal(value);
+                            where[param] = value;
                         }
```

Comment:
We could maybe simplify this to:

where[param] = Array.isArray(value) ? Between(value[0], value[1]) : Equal(value);